### PR TITLE
chore(functions): update node-fetch invocation

### PR DESCRIPTION
**Pull request by ace-n**

### CODE REVIEW

**Review by fhinkel — APPROVED** [review]

File: functions/tips/index.js [hunk 1]
```diff
@@ -94,8 +94,15 @@ const httpsAgent = new https.Agent({keepAlive: true});
  */
 exports.connectionPooling = async (req, res) => {
   try {
-    const {data} = await fetch('/', {httpAgent, httpsAgent});
-    res.status(200).send(`Data: ${data}`);
+    const url = 'https://www.example.com/';
```
Comment (grant) [inline functions/tips/index.js:97]: Can we add a TODO here? A developer has to replace this URL.

File: functions/tips/index.js [hunk 2]
```diff
@@ -94,8 +94,15 @@ const httpsAgent = new https.Agent({keepAlive: true});
...
+
+    // Select the appropriate agent to use based on the URL.
+    const agent = url.includes('https') ? httpsAgent : httpAgent;
```
Comment (grant) [inline functions/tips/index.js:100]: This constant could probably be removed by inlining it.

**Review by grant — APPROVED** [review]
LGTM. Some comments that could improve the PR.

File: functions/tips/index.js (hunk 2 above)
Comment (ace-n) [reply functions/tips/index.js:100]: I wanted to call it out explicitly, which is why I didn't do that.

File: functions/tips/index.js (hunk 1 above)
Comment (ace-n) [reply functions/tips/index.js:97]: Not necessarily - the URL provided will work as-is.

That said, I'll add an optional TODO.

**Pull request merged**